### PR TITLE
Cards

### LINKED ISSUE #7: Display Zones in cards on the dashboard
**Blocked by** #6 

As a user I want to be able to see my created zones on the dashboard so that I can have a quick overview of the status on my property.

**Note:** The values can be hardcoded for now, but it would be great if we could already connect it to hardcoded (fake) reactive Vue data, so that it will be easier to switch over to real api data. I'm happy to assist with this part.

**Criteria:**

- Each card present the following data: 
   - Zone name  
   - Current temperature
   - Current humidity
   - Highest temperature in the last 24 hours
   - Lowest temperature in the last 24 hours
 - Bonus data (if the card doesn't get too cluttered)
   - Ideal temperature
   - Ideal humidity

### PR DESCRIPTION
**Pull request by AndreaOsk**
Fixes #7 

Here's what I've been working on over the last few days. 

Next stage will be to address the open Issues. 
I need to do a bit more research.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,22 +1,22 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png" />
-
+  <!-- <img alt="Vue logo" src="./assets/logo.png" /> -->
```
Comment (vilbergs) [inline src/App.vue]: This can be removed, we won't need the Vue logo in the future.

**Review by vilbergs — CHANGES_REQUESTED** [review]
Awesome!

I love that a card component is coming together. However it would be fantastic if it could fit the criteria in #6. Alternatively this could be the component described in #7, but then it would be great to rename it to say, ZoneCard. We can then refactor it later when we have a generic card component in place.

**Comment by AndreaOsk** [comment]
If I merge will the change be applied or should I fix and create a new PR?

File: src/App.vue (hunk 1 above)
Comment (AndreaOsk) [reply src/App.vue]: This has now been removed.

File: src/components/ZoneCard.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  title: String
+});
+
+const state = reactive({
+  Temperature: 40,
+  Humidity: 32,
+  maxTemp: 41,
+  minTemp: 12,
+  maxHumid: 43,
+  minHumid: 28
```
Comment (vilbergs) [inline src/components/ZoneCard.vue]: The casing in the object properties (keys) is a little inconsistent here. The general convention is to use `lowerCamelCase`

**Review by vilbergs — COMMENTED** [review]
I took the liberty to resolve some conflicts after I merged my branch. Hopefully these will be less common when we've established the component architecture better.

File: src/components/ZoneCard.vue (hunk 2 above)
Comment (AndreaOsk) [reply src/components/ZoneCard.vue]: Yes, you are right. I was aware of this and should have fixed this already.

File: src/views/Dashboard.vue [hunk 3]
```diff
@@ -1,10 +1,14 @@
 <template>
   <div>
     <h1>Dashboard!</h1>
+    <div>
+      <ZoneCard title="Sovrum" />
```
Comment (AndreaOsk) [inline src/views/Dashboard.vue:5]: Ahh, I like this. Much neater.

**Comment by AndreaOsk** [comment]
@vilbergs we're having some conflicts with merging the branches. Could you take a look and advise.

**Review by vilbergs — APPROVED** [review]

**Pull request merged**